scripts/testgrid_analysis/cmd/data.go

```diff
@@ -0,0 +1,138 @@
+// Copyright 2024 The etcd Authors
```

Comment:
Thanks for working on this @siyuanfoundation!

My instinct would be the golang utility would be better suited to live under `tools/`?

Reply:
Suggest to put it under https://github.com/etcd-io/etcd/tree/main/tools

Reply:
moved to tools